server/internal/adapter/integration/item_api.go

```diff
@@ -87,7 +87,7 @@ func (s Server) ItemCreate(ctx context.Context, request ItemCreateRequestObject)
 		return nil, err
 	}
 
-	return ItemCreate200JSONResponse(toItem(i, ver[len(ver)-1], id.NewModelID())), nil
+	return ItemCreate200JSONResponse(toItem(i, ver[len(ver)-1], mId)), nil
```

Comment:
please fix `toItem` func to use item's modelID without model ID from outside: `item.Model()`